api/src/db_operations.py

```diff
@@ -17,23 +17,23 @@
 from models import *
 from app import create_app
 
-def reset():
-    drop_tables()
-    create_tables()
-    seed_db()
+def reset(config):
+    drop_tables(config)
+    create_tables(config)
+    seed_db(config)
 
-def drop_tables():
-    app = create_app('dev')
+def drop_tables(config):
```

Comment:
Should we consider setting the default value of ```config``` to ```'dev'``` for backwards compatibility? I'm not sure if this gets called anywhere, or if it was just a shell script.

Reply:
I don't think it affects anything else from this change but not 100% sure. But making the default value be 'config=dev' is a good idea! I'll change it to that. I think I have time to get this pushed this morning.